### ISSUE TITLE
Add LICENSE to each colab.

### ISSUE THREAD

**Issue by reddragon** [EfficientDL/book#5]
As per title.

**Comment by reddragon** [comment]
We need to figure out which license to use though. Is Apache the right one?

**Comment by NareshPS** [comment]
Let's figure it out before putting a license. It's not a good idea to
change it later.

**Comment by NareshPS** [comment]
We need separate licenses for book pdf and code labs. Book PDFs should have a limited license in terms of reproducibility. We want to discourage people from publishing these chapters. Code labs, however, can have a free to use, modify license with attribution.

**Comment by NareshPS** [comment]
For the code labs we can use MIT License. It seems to fit our use case. Apache license requires the users to state the changes made to the code. I am kinda on the fence. It may be a good idea for some cases while too cumbersome for other cases. Take a look at the common licenses here if you haven't already: https://choosealicense.com/licenses/

I am still looking for a license to the PDFs. My current disposition is to avoid having a license for it. I don't know a use case for the readers to reproduce the book text. Readers should attribute the original material based on their usage.

**Comment by reddragon** [comment]
I agree, I was mostly thinking of license for code only. For book I guess it might depend on the potential publisher.

**Comment by NareshPS** [comment]
Let's leave it open for a couple of days just in case we have other ideas and add the license later this week.
